mistralrs-core/src/pipeline/macros.rs

```diff
@@ -2,14 +2,85 @@
 #[macro_export]
 macro_rules! api_dir_list {
     ($api:expr, $model_id:expr) => {
-        $api.info()
-            .map(|repo| {
-                repo.siblings
-                    .iter()
-                    .map(|x| x.rfilename.clone())
-                    .collect::<Vec<String>>()
-            })
-            .unwrap_or_else(|e| {
+        if std::path::Path::new($model_id).is_absolute() {
```

Comment:
Can we just check if the path exists? For example, someone may pass `../models/gemma2`.

Reply:
Sure, I think maybe this new change is better and more simple🤣